src/components/hooks.ts

```diff
@@ -0,0 +1,6 @@
+export function useFormatedDate (date: Date | string): string {
+  const dateTime = new Date(date)
+  const month = dateTime.toLocaleString('default', { month: 'long' })
+  const year = dateTime.getFullYear()
+  return `${month} ${year}`
+}
```

Comment:
```suggestion
const useFormattedDateRange = ({ dateFrom, dateTo }) => {
...
[... je te laisse chercher ...]
...
  return {
    formattedDateFrom,
    formattedDateTo,
})
```

Comme ca tu pourra faire 
```
const {
  formattedDateFrom,
  formattedDateTo,
} = useFormattedDateRange(props)
```